unstable/wlr-layer-shell-unstable-v1.xml

```diff
@@ -203,21 +204,85 @@
       <arg name="left" type="int"/>
     </request>
 
+    <enum name="keyboard_interactivity" since="4">
+      <description summary="types of keyboard interaction possible for a layer shell surface">
+        Types of keyboard interaction possible for layer shell surfaces. The
+        rationale for this is twofold: (1) some applications are not interested
+        in keyboard events and not allowing them to be focused can improve the
+        desktop experience; (2) some applications will want to take exclusive
+        keyboard focus.
+      </description>
+
+      <entry name="no_focus" value="0">
```

Comment:
Bikeshedding time: maybe `none` is a better name?

Reply:
Yes, sure